infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -670,7 +679,7 @@ private void correctOutOfPhaseEpochs(PollReport pollReport) {
         }
 
         try {
-            Layout layout = serverContext.getManagementLayout();
+            Layout layout = getManagementLayout();
             // Query all layout servers to get quorum Layout.
             Map<String, CompletableFuture<Layout>> layoutCompletableFutureMap = new HashMap<>();
             for (String layoutServer : layout.getLayoutServers()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "layout" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)